genericx86-64.coffee

```diff
@@ -0,0 +1,57 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+GENERIC_FLASH = '''
+	Power up the <%= TYPE_NAME %>. Make sure you have a keyboard connected.
+	Press the F10 key (may differ on some platforms) while BIOS is loading in order to enter the boot menu. 
+  Next, select the name of your USB key.
+'''
+
+GENERIC_POWERON = 'Power on your device.'
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	GENERIC_POWERON
+]
+
+module.exports =
+	version: 1
+	slug: 'genericx86-64'
+	aliases: [ 'nuc', 'intel-nuc' ]
+	name: 'Generic x86-64'
+	arch: 'amd64'
+	state: 'released'
+
+	stateInstructions:
+		postProvisioning: postProvisioningInstructions
+
+	instructions: [
+		instructions.ETCHER_USB
+		instructions.EJECT_USB
+		instructions.FLASHER_WARNING
+		GENERIC_FLASH
+	].concat(postProvisioningInstructions)
+
+	gettingStartedLink:
+		windows: 'http://docs.resin.io/nuc/nodejs/getting-started/#adding-your-first-device'
+		osx: 'http://docs.resin.io/nuc/nodejs/getting-started/#adding-your-first-device'
+		linux: 'http://docs.resin.io/nuc/nodejs/getting-started/#adding-your-first-device'
```

Comment:
These are supposed to be changed I guess. Sync with @zwalchuk on the link and change as needed (even if the links do not currently exist)

Reply:
@zwalchuk the nuc part of the link should be replaced with genericx86-64.

Reply:
@zwalchuk can you let @telphan how the links should look like so that he can change this PR accordingly?

Reply:
@telphan, the links should be https://docs.resin.io/genericx86-64/nodejs/getting-started/#add-your-first-device. And please create a ticket in the docs repo requesting a getting started guide for this device type.